apps/web/components/dialog/EditLocationDialog.tsx

```diff
@@ -382,7 +382,7 @@ export const EditLocationDialog = (props: ISetLocationDialog) => {
                 }}
               />
               {selectedLocation && LocationOptions}
-              <DialogFooter className="mt-4">
+              <DialogFooter className="!static mt-4">
```

Comment:
is there any other way we can fix the error? this looks like a hacky solution

Reply:
We can add overflow: auto property on the form tag which can help us fix the button overflow on the text field. This way we wont need to pass !static on the DialogFooter component.

![Screenshot 2023-12-06 175640](https://github.com/calcom/cal.com/assets/77525994/af1f079c-f29f-40aa-ac76-b0d150aba0da)